client/client_test.go

```diff
@@ -1327,5 +1423,10 @@ func TestOpenFgaClient(t *testing.T) {
 		if err != nil {
 			t.Fatalf("%v", err)
 		}
+		// WriteAssertions without options should work
+		_, err = fgaClient.WriteAssertions(context.Background()).Body(requestBody).Execute()
+		if err != nil && err.Error() != "Required parameter AuthorizationModelId was not provided" {
```

Comment:
Same as above, do we ever expect 'Required parameter AuthorizationModelId was not provided'?